engine-tests/src/test/java/org/terasology/utilities/collection/ListOrderedSetTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.utilities.collection;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ListOrderedSetTest {
```

Comment:
@msteiger: Martin, can you double-check all is well in this class? I have no reason to believe otherwise but I'm not up to speed with Unit testing goals and standards.